server/init.go

```diff
@@ -12,16 +12,27 @@ import (
 // phrase to recover the private key.
 func GenerateCoinKey(algo keyring.SignatureAlgo) (sdk.AccAddress, string, error) {
 	// generate a private key, with recovery phrase
-	info, secret, err := keyring.NewInMemory().NewMnemonic("name", keyring.English, sdk.GetConfig().GetFullBIP44Path(), keyring.DefaultBIP39Passphrase, algo)
+	info, secret, err := keyring.NewInMemory().NewMnemonic(
+		"name",
+		keyring.English,
+		sdk.GetConfig().GetFullBIP44Path(),
+		keyring.DefaultBIP39Passphrase,
+		algo)
 	if err != nil {
-		return sdk.AccAddress([]byte{}), "", err
+		return sdk.AccAddress{}, "", err
 	}
+
 	return sdk.AccAddress(info.GetPubKey().Address()), secret, nil
 }
 
 // GenerateSaveCoinKey returns the address of a public key, along with the secret
 // phrase to recover the private key.
-func GenerateSaveCoinKey(keybase keyring.Keyring, keyName string, overwrite bool, algo keyring.SignatureAlgo) (sdk.AccAddress, string, error) {
+func GenerateSaveCoinKey(
+	keybase keyring.Keyring,
+	keyName, mnemonic string,
```

Comment:
Ah, this is technically API-breaking and shouldn't have been backported.

Who's really needing this change? If yes, then I'll put some big signs on the Release Notes; if not, then we can revert it?

Reply:
Sounds like we should move this function to testutils, it's only used in tests.

Reply:
For a non-breaking change, we can create a new `GenerateSaveCoinKeyWithMnemo()` for 0.44 and 0.45

Reply:
tbh, I would move this function away to testutils in 0.45

Reply:
Moving this to `testutils` seems like a good approach. I would like to see these in 0.44 and 0.45 regardless, so ACK on @robert-zaremba's proposal 👍